File: Java&Kotlin lecture/src/main/java/com/java/many/Array.java
package com.java.many;

public class Array {
    public static void main(String[] args) {
        int[] numbers = new int[5];
        numbers[0] = 1;
        numbers[4] = 5;

        int[] scores = {66, 77, 88, 99, 50};
        System.out.println(scores);

        for(int i = 0; i < scores.length; i++) {
            System.out.println(scores[i]);
        }

        for(int score : scores) {
            System.out.println(score);
        }
    }
}

File: Java&Kotlin lecture/src/main/kotlin/com/kotlin/many/Array.kt
package com.kotlin.many

fun main(args: Array<String>) {
    val numbers = IntArray(5)
    numbers[0] = 1
    numbers[4] = 5

    val scores = intArrayOf(66, 77, 88, 99, 50)
    println(scores)

    for (i in 0 .. scores.size-1) {
        println(scores[i])
    }

    for (i in scores.indices) {
        println(scores[i])
    }

    for (score in scores) {
        println(score)
    }
}